<launch>
    
    <!-- Spawn Task-2 Models in Gazebo -->
    <include file="$(find pkg_vb_sim)/launch/task5_world.launch" />

    <!-- Spawn two UR5 in Gazebo -->
    <include file = "$(find pkg_vb_sim)/launch/two_ur5_gazebo.launch" />

    <!-- Spawn T5 Boxes -->
    <rosparam file ="$(find pkg_vb_sim)/config/config_package_colour.yaml"/>
    <node name= "task5_spawn_models" pkg= "pkg_vb_sim" type="task5_spawn_models.py"/>

    <!-- Run Conveyor Belt Service Server -->
    <node name= "node_service_server_conveyor_belt" pkg= "pkg_vb_sim" type="node_service_server_conveyor_belt.py" output="screen"/>

    <!-- Run the Online Order Placer Node -->
    <rosparam file ="$(find pkg_vb_sim)/config/config_online_order.yaml"/>
    <node name= "node_online_order_placer" pkg= "pkg_vb_sim" type="node_online_order_placer.py" output="screen"/>
	
</launch>